bindata/assets/config/defaultconfig.yaml

```diff
@@ -163,6 +163,8 @@ apiServerArguments:
     - 70s # give SDN some time to converge: 30s for iptable lock contention, 25s for the second try and some seconds for AWS to update ELBs
   shutdown-send-retry-after:
   - "true"
+  shutdown-watch-termination-grace-period:
+    - 25s
```

Comment:
does the server drain watch requests, or does it wait for them to terminate naturally?

Reply:
ah, ok, it rate limits # of watchers that will be closed. 

does setting this flag affect (slow down) the shutdown process? I just want to make sure no other adjustments are required.